### PR TITLE
Added new test for product name

### PR DESCRIPTION
**Pull request by psav**
Purpose or Intent
=================

* Product name is now obtained from the ymls
* Product name is now dynamically inserted

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 109e295cf9555ebb390b5bffa4f8f82356b3069e:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 96073e9829fde52d5de708975f7ae9d951619c84:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/configure/configuration/__init__.py [hunk 1]
```diff
@@ -178,10 +178,8 @@ def server_zone_description():
 
 
 def server_region_string():
-    if store.current_appliance.is_downstream:
-        return "CFME Region: Region {} [{}]".format(*server_region_pair())
-    else:
-        return "ManageIQ Region: Region {} [{}]".format(*server_region_pair())
+    return "{} Region: Region {} [{}]".format(
```
Comment (psav) [inline cfme/configure/configuration/__init__.py]: This code will work after 5.7.0.4, so we'll just leave this PR open till the next build

**Comment by dajoRH** [comment]
Lint report for commit 4f263d01d5eef952755d504ebeb694e2dc4361c6:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**